model/src/main/java/org/cloudfoundry/identity/uaa/zone/Links.java

```diff
@@ -98,25 +98,35 @@ public Logout setWhitelist(List<String> whitelist) {
     }
 
     public static class SelfService {
-        private boolean selfServiceResetPasswordEnabled = true;
-        private boolean selfServiceCreateAccountEnabled = true;
+        private boolean selfServiceLinksEnabled = true;
```

Comment:
why we added selfServiceLinksEnabled back and converted type of ther flags to String?

Reply:
Compatibility for existing zones, selfServiceLinksEnabled field has been added back.